applications/ExaquteSandboxApplication/custom_python/exaqute_sandbox_python_application.cpp

```diff
@@ -43,7 +43,8 @@ PYBIND11_MODULE(KratosExaquteSandboxApplication,m)
     AddCustomProcessesToPython(m);
 
     //registering variables in python
-      KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, DIVERGENCE_WEIGHTED )
+      KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, AVERAGED_DIVERGENCE )
+      KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, DIVERGENCE )
```

Comment:
There's already a `DIVERGENCE` variable in the FluidDynamicsApplication. I'm wondering if it would make sense to migrate it to the core.

Reply:
I think it is a good idea, I would propose to open a PR to do this after merging this PR.